deploy/src/deploymentUtils.js

```diff
@@ -10,23 +10,27 @@ const {
   FOREIGN_RPC_URL,
   HOME_RPC_URL,
   GAS_LIMIT,
-  GAS_PRICE,
+  HOME_GAS_PRICE,
+  FOREIGN_GAS_PRICE,
```

Comment:
There are already variables with names `HOME_GAS_PRICE` and `FOREIGN_GAS_PRICE` and they have different meaning (take a look at `deploy/src/native_to_erc/home.js`, `deploy/src/erc_to_erc/home.js`, `deploy/src/native_to_erc/foreign.js` and `deploy/src/erc_to_erc/foreign.js`). In order to avoid confusion in the future, could we assign other names to variables used to deploy/configure contracts?

Reply:
I have added such kind of changes in #87 (https://github.com/poanetwork/poa-bridge-contracts/pull/87/commits/0627391ea1827667dc66b856ba89071120c47342). You can review suggested solution and cherry-pick this commit to your branch.

Reply:
I applied those changes, thanks.